react/src/pages/utils/components/ChipTransferList.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { Box, Chip, Grid, makeStyles, Paper } from "@material-ui/core";
+import { PersonAdd, PersonAddDisabled } from "@material-ui/icons";
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        margin: theme.spacing(1),
+    },
+}));
+
+export interface TransferChip {
+    label: string;
+    key: number;
+    selected: boolean;
+    content?: unknown;
+}
+
+/**
+ * A chip transfer list. Users can click on chips to
+ * move them between the two lists.
+ */
+export default function ChipTransferList(props: {
+    chips: TransferChip[];
+    setChips: (chips: TransferChip[]) => void;
+}) {
+    const classes = useStyles();
+
+    function toggleSelected(chip: TransferChip) {
+        props.setChips(
+            props.chips.map(item =>
+                item.key === chip.key ? { ...item, selected: !item.selected } : item
+            )
+        );
+    }
+
+    return (
+        <Grid container xs={12} spacing={1} className={classes.root}>
```

Comment:
```suggestion
        <Grid container spacing={1} className={classes.root}>
```
The `xs` prop only affects `Grid item`